app/controllers/bourbons_controller.rb

```diff
@@ -4,7 +4,14 @@ class BourbonsController < ApplicationController
   before_action :authorize_admin!, only: [:destroy]
 
   def index
-    @bourbons = Bourbon.page(params[:page])
+    if params[:search]
+      @bourbons = Bourbon.search(params[:search])
+      .order("name")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression in an assignment spanning multiple lines.